.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+name: Fix Code Style
+ 
+on: [push]
+ 
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [8.3]
+ 
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+ 
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: json, dom, curl, libxml, mbstring
+          coverage: none
+ 
+      - name: Install Pint
+        run: composer global require laravel/pint
+        working-directory: ./src/backend
+ 
+      - name: Run Pint
+        run: pint
+        working-directory: ./src/backend
+ 
+      - name: Commit linted files
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "Fixes coding style"
```

Comment:
I know this is the way that is given in the Pint documentation, but I'm not a huge fan of this. Could we instead only run `pint` is check mode? I.e. just have the CI fail, and leave it to developers to actuallly fix the lint errors?